src/app/components/course/CourseDetails/CourseDetailsTopBanner/CourseDetailsTopBanner.tsx

```diff
@@ -19,18 +20,20 @@ const CourseDetailsTopBanner: React.FC = () => {
     <div className={classNames('row', styles.topBannerMainContainer)}>
       <div
         className={classNames(
-          { 'col-md-7': windowWidth > 768 },
-          { 'col-md-12': windowWidth <= 820 }
+          { 'col-md-7': windowWidth > 1000 },
+          { 'col-md-12': windowWidth <= 1000 }
```

Comment:
The purpose of using `col-md-` classes is that they are responsive. When you say `col-md-7`, it means that for medium screens and above that, the content will encompass 7/12th of the container size. If the screen is less than the size of a medium screen, it will cover the whole container's width (not only 7/12th). Try to think of a way to use only one class here, without conditionals.